tests/pkg_arduino_sdi_12/Makefile

```diff
@@ -1,10 +1,15 @@
 include ../Makefile.tests_common
 
+ifneq (,$(filter remote-revb,$(BOARD)))
```

Comment:
What's special about the `remote-revb` here?
Why can't we always use the `cpp11-compat` and make this work on every board?

Reply:
Because then it fails for the Arduino boards:

```
$ BOARD=arduino-mega2560 make -C tests/pkg_arduino_sdi_12
make: Entering directory '/home/jdavid/sandboxes/UiO/RIOT/tests/pkg_arduino_sdi_12'
There are unsatisfied feature requirements: libstdcpp
```